From 281bae2b5cf76703b328925b85dbde110d47ac79 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 4 Oct 2024 18:37:12 +0200
Subject: [PATCH 1392/1695] plat-stm32mp2: fix power level description on
 stm32mp25

Fix a minor error in power level description for STM32M25 family.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git cc9726ceb00e46195f543d46d6e2310b792b35ac plat-stm32mp2: fix power level description on stm32mp25]
Fixes: 0d98c255fb4d ("plat-stm32mp2: add pm support on stm32mp25")
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ibe623dcc4b3f3a52b61cc61ba4ae0bc73ca3e07f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/409249
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/stm32mp_pm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp2/stm32mp_pm.h b/core/arch/arm/plat-stm32mp2/stm32mp_pm.h
index d8ac02dc4..c56fbafe7 100644
--- a/core/arch/arm/plat-stm32mp2/stm32mp_pm.h
+++ b/core/arch/arm/plat-stm32mp2/stm32mp_pm.h
@@ -14,7 +14,7 @@
  * 0: CPU1 core#0 or core#1   (Stop1 or LP-Stop1)
  * 1: D1 domain               (LPLV-Stop1)
  * 2: LPLV D1                 (Stop2 or LP-Stop2)
- * 3: D2                      (LPLV-Stop1)
+ * 3: D2                      (LPLV-Stop2)
  * 4: LPLV D2                 (Standby)
  * 5: MAX                     (PowerOff)
  *
-- 
2.47.2

